src/midcom/datamanager/template/view.php

```diff
@@ -65,7 +65,7 @@ public function form_row(FormView $view, array $data)
         $string = '<div class="' . $class . '">';
         $string .= $this->renderer->label($view);
         $string .= '<div class="value">';
-        $string .= $this->renderer->widget($view);
+        $string .= $this->renderer->humanize($this->renderer->widget($view));
```

Comment:
See comment above. Youe translating HTML here, which doesn't seem like it would work